x-pack/solutions/security/plugins/security_solution/public/detections/components/attacks/table/attack_details/alerts_tab.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Filter } from '@kbn/es-query';
+import React from 'react';
+
+import { TableId } from '@kbn/securitysolution-data-table';
+import { PageScope } from '../../../../../data_view_manager/constants';
+import { AlertsTable } from '../../../alerts_table';
+
+/** Test subject constant for AlertsTab component */
+export const ALERTS_TAB_TEST_ID = 'alertsTab';
+
+interface Props {
+  /** Filters applied from grouping */
+  groupingFilters: Filter[];
+  /** Default filters to apply to the alerts table */
+  defaultFilters: Filter[];
+  /** Whether the alerts table is in a loading state */
+  isLoading: boolean;
+}
+
+/**
+ * Component that displays the alerts tab content, rendering an AlertsTable
+ * with filters for the specific attack's alerts.
+ */
+export const AlertsTab = React.memo<Props>(({ groupingFilters, defaultFilters, isLoading }) => {
+  return (
+    <div data-test-subj={ALERTS_TAB_TEST_ID}>
+      <AlertsTable
+        tableType={TableId.alertsOnAttacksPage}
+        inputFilters={[...defaultFilters, ...groupingFilters]}
```

Comment:
Also one advantage of having this merging done in a higher up component, is those components are memoized. Currently with the code you have, the ResponseOps alerts table will keep being rerendered because we keep passing a new array every time